app/views/photos/edit.html.erb

```diff
@@ -1,41 +1,7 @@
 <h1><%= action_name.capitalize %> Photo</h1>
-<div class="form-container">
-    <%= form_with(model: @photo, local: true) do |form| %>
-        <% if @photo.errors.any? %>
-            <div id="error_explanation">
-                <h2><%= pluralize(@photo.errors.count, "error") %> prohibited this photo from being saved:</h2>
-                <ul>
-                    <% @photo.errors.full_messages.each do |message| %>
-                        <li><%= message %></li>
-                    <% end %>
-                </ul>
-            </div>
-        <% end %>
 
-    <div class="field">
-        <%= form.label :Photo_Link %>
-        <%= form.text_field :photoLink %>
-    </div>
+<p> <%= @photo.url %> </p>
```

Comment:
`<p> <a href="<%= photo.url %>" target="_blank"></a></p>`
maybe change photo url to a hyperlink instead

Reply:
Done.